chromium_src/components/viz/service/gl/gpu_service_impl.h

```diff
@@ -12,7 +12,7 @@
 #define InitializeWithHost                                             \
   InitializeWithHost_ChromiumImpl(                                     \
       mojo::PendingRemote<mojom::GpuHost> pending_gpu_host,            \
-      gpu::GpuProcessActivityFlags activity_flags,                     \
+      gpu::GpuProcessShmCount activity_flags,                          \
```

Comment:
Should change to upstream var name `use_shader_cache_shm_count`.

Reply:
Fixed in ade597473d9f5a7808d15ff676e8c3db292a4529.